airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/cdc/MssqlDebeziumStateUtil.java

```diff
@@ -125,7 +125,7 @@ public JsonNode constructInitialDebeziumState(final Properties properties,
     assert Objects.nonNull(schemaHistory.schema());
 
     final JsonNode asJson = serialize(offset, schemaHistory);
-    LOGGER.info("Initial Debezium state constructed: {}", asJson);
+    LOGGER.debug("Initial Debezium state constructed: {}", asJson);
```

Comment:
Do we still want to log this similar to here : https://github.com/airbytehq/airbyte/pull/35565/files#diff-3736b2b724ebf28c50445ea0c188567b55b299a2d52a3f7e0a43b307108ef251

to still keep it an info log?